docs/docs/features/manifest.md

```diff
@@ -20,8 +20,6 @@ Choose this approach for deployments with a well-defined scope, but a possibly l
 
 ## Fully updatable deployment
 
-<EnterpriseBanner/>
```

Comment:
you can also remove `docs/src/components/enterprise-banner.md` and the two lines in `docs/src/theme/MDXComponents.js`